exercises/concept/inventory-management/t/inventory-management.t

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env perl
+
+use v5.38;
+use Test2::V0;
+
+use InventoryManagement ();
+
+subtest 'create_inventory($items)' => sub { # begin: create task: 1
+    is(
+        InventoryManagement::create_inventory( [ 'wood', 'iron', 'iron', 'diamond', 'diamond' ] ),
```

Comment:
Space inside argument list delimiters, and around the list delimiter?

Reply:
Perltidy is responsible for this. I think it should be left as is and tweaked via perltidy settings later to adjust.

Reply:
Sounds good.  (Less painful as well since we can do that magic marker mentioned in [no important files changed flag] post.)

[no important files changed flag]: https://forum.exercism.org/t/maintainers-new-no-important-files-changed-flag-added/6500